tm-bench/transacter.go

```diff
@@ -124,13 +125,19 @@ func (t *transacter) sendLoop(connIndex int) {
 			for i := 0; i < t.Rate; i++ {
 				// each transaction embeds connection index and tx number
 				tx := generateTx(connIndex, txNumber)
+				paramsJson, err := json.Marshal(map[string]interface{}{"tx": hex.EncodeToString(tx)})
```

Comment:
should we use tendermint#WSClient here (with reconnects/pingpongs) ? like if there is a reconnect, will the numbers be correct? i.e. we claim we sent 10000 txs, but due to reconnects only 5000 made it to the server